platform_versioned_docs/version-24.2/enterprise/advanced-topics/manual-azure-batch-setup.mdx

```diff
@@ -0,0 +1,487 @@
+1. In the Azure Portal, go to the Batch account you created earlier.
+1. Go to the **Pools** tab and find the pool called `tower-pool-${id}`, where `${id}` is the ID of the head node pool created earlier.
+1. Select **Identity**.
+1. Select **Add User Assigned Identity**.
+1. Select the managed identity created earlier.
+1. Select **Add**.
+
+Processes running on this pool can now use the managed identity to authenticate to Azure Batch and Storage.
+
+In Seqera:
+
+1. Add a new compute environment with the name `entra-mi` and select the Azure Batch **Provider** type.
+1. For **Location**, select the same region as your Batch account.
+1. For **Config mode**, select Manual.
+1. For **Compute pool**, select the pool you added the managed identity to earlier (`tower-pool-${id}`).
+1. For **Managed Identity Client ID**, enter the client ID of the managed identity created earlier.
+
+Duplicate the `hello-world-autopool-low-priority` pipeline and save it as `hello-world-entra-mi`.
+
+Select **Launch** next to the `hello-world-entra-mi` pipeline in your workspace Launchpad to complete the launch form and launch the workflow.
+
+The pipeline will run as before, but using the managed identity to authenticate to Azure Batch and Storage. No keys or storage required.
+
+:::note
+You can also use User Subscription mode instead of Batch Managed here, but this is beyond the scope of this tutorial.
+:::
+
+Behavior:
+
+- Seqera authenticates to Azure Batch and Azure Storage using a service principal.
+- It submits a job and task to the Azure Batch service using the service principal.
+- The task runs Nextflow, which authenticates to Azure Batch and Azure Storage using the managed identity.
+- All processes run on the head node as in the first example.
+
+Advantages:
+
+- No keys or short access tokens are exchanged, increasing security.
+- A service prinicipal can have very granular permissions, so you can grant it only the permissions it needs.
+- Managed identities can be scoped to a specific resource, so the Nextflow head job has very restricted permissions.
+- Different managed IDs can have different permissions, so different compute environments can have different scoped permissions.
+
+Disadvantages:
+
+- The setup is quite complicated with room for error.
+- Errors can be harder to troubleshoot.
+
+### 7. Use a node pool attached to a VNet
+
+It is common to attach Azure Batch pools to a virtual network. This is useful to connect to other resources in the same VNet or place things behind enhanced security. Seqera Platform does not support this feature directly, so you must manually create an Azure Batch pool.
+
+See [Create a pool with a Virtual Network in the Azure portal](https://learn.microsoft.com/en-us/azure/batch/batch-virtual-network#create-a-pool-with-a-virtual-network-in-the-azure-portal) to create an Azure Batch pool manually that is compatible with Seqera and Nextflow. Use the following settings:
```

Comment:
@adamrtalbot This URL was still TODO before. Let me know if this docs link I plugged in is the appropriate one.